src/main/java/StringManipulation.java

```diff
@@ -1,33 +1,113 @@
+import java.util.*;
+import java.lang.String;
+
+
 public class StringManipulation implements StringManipulationInterface {
 
+    private String manipulatedstring;
     @Override
     public String getString() {
-        return null;
+        return manipulatedstring;
     }
 
     @Override
-    public void setString(String string) {
-    }
+    public void setString(String string) { manipulatedstring = string;}
```

Comment:
Ok I have noted that your data member for the string in question is "manipulatedstring" this is a good name and place to begin